src/app/domain/osReceive/models/OsReceiveCozyApp.ts

```diff
@@ -55,9 +56,16 @@ export interface AppForUpload {
   slug: string
 }
 
-export const fetchOsReceiveCozyApps = {
-  definition: Q('io.cozy.apps'),
-  options: {
-    as: 'io.cozy.apps/accept_from_flagship'
+export const fetchStackApp = (
```

Comment:
our convention is to name that : 

```
buildXXXQuery()
```

And then use it like that: 
```
const appQuery = buildAppQuery(slug)
const {data: apps, fetchStatus } = useQuery(appQuery.definition, appQuery.options)
```

`fetchXXX` should only be used when we make the call. Not the step before.